source/libs/wal/inc/walInt.h

```diff
@@ -66,6 +67,12 @@ static inline int64_t walGetLastFileSize(SWal* pWal) {
   return pInfo->fileSize;
 }
 
+static inline int64_t walGetLastFileCachedSize(SWal* pWal) {
```

Comment:
Compute the data not-synced still since last save of meta and files(i.e. with fsync)